packages/backend-function/src/function_env_translator.ts

```diff
@@ -9,8 +9,10 @@ import * as iam from 'aws-cdk-lib/aws-iam';
  * in order to resolve secrets in environment props.
  */
 export class FunctionEnvironmentTranslator {
-  private secretPaths: string[] = [];
+  private ssmPaths: string[] = [];
```

Comment:
I made some of the terms in this file more generic since we are handling any type of ssm param in this file